### PR TITLE
Fast endpoints: chatbot

### LINKED ISSUE #57: save the registration of the volunteer from tg-bot
- check if the user with the telegram_id exists
- if not, create a new volunteer with is_active = false, set the chat-id-telegram

### PR DESCRIPTION
**Pull request by alexciornii**
Closes #57 #58 #62 #73

### CODE REVIEW

File: backend/endpoints/volunteer.py [hunk 1]
```diff
@@ -1,33 +1,77 @@
...
-            new_volunteer["password"] = PassHash.hash(new_volunteer["password"])
-            assert not Volunteer.objects(email=new_volunteer['email']) , "user with this email already exists"
-            new_volunteer['created_by'] = created_by#g.user.get().clean_data()['_id']
-            comment = Volunteer(**new_volunteer)
-            comment.save()
-            return jsonify({"response": "success", 'user': comment.clean_data()})
-        except Exception as error:
-            return jsonify({"error": str(error)}), 400
+
+        if created_by == os.getenv("COVID_BACKEND_USER"):
```
Comment (johnsmithm) [inline backend/endpoints/volunteer.py:40]: it limits the availability of the COVID_BACKEND_USER to add volunteers from UI, the tg-bot request can be identified using chat-id

Comment (alexciornii) [reply backend/endpoints/volunteer.py:40]: I think this is done.

**Comment by johnsmithm** [comment]
phonenumbers library is not added to the dockerfile

**Comment by alexciornii** [comment]
> phonenumbers library is not added to the dockerfile

Done

**Comment by johnsmithm** [comment]
- vu and os is not defined when adding new volunteer using tg-bot
- name_phone is not working as needed, __gt is not enough, here is an idea
```

nr = len(phone_name)
                    phone_name = int(phone_name) % 10000000
                    phone_name_ = phone_name+1
                    obj = Beneficiary.objects(phone__gt=phone_name*10**(8-nr), phone__st=phone_name_*10**(8-nr))
```

**Comment by johnsmithm** [comment]
also the tg-bot is sending the json from bellow when creating a new volunteer, need to be changed to the volunteer requirements: phone-8digits, telegram_id and telegram_chat_id values.

```
{
      "first_name": "Ion", 
      "last_name": "I2on", 
      "availability":2,
      "activities": ["ee"],
      "phone": "+37368628335",
      "email": "ee@ss.rr",
      "chat_id":414370920,
      "phoneEx": 40723405038
     
    }
```

**Pull request merged**